packages/remix-server-runtime/data.ts

```diff
@@ -49,7 +49,9 @@ export async function callRouteAction({
     request: stripDataParam(stripIndexParam(request)),
     context: loadContext,
     params,
-    response,
+    // Only provided when single fetch is enabled, and made available via
+    // `defineAction` types, not `ActionFunctionArgs`
+    ...(singleFetch ? { response } : null),
```

Comment:
This is functionally equivalent, as `response` is only defined when `singleFetch` is enabled, but this quiets the type error when response is not in `LoaderFunctionArgs`